analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Extensions/IOperationExtensions.cs

```diff
@@ -61,9 +61,6 @@ internal static ISymbol TrackedSymbol(this IOperation operation)
     internal static IIsPatternOperationWrapper? AsIsPattern(this IOperation operation) =>
         operation.As(OperationKindEx.IsPattern, IIsPatternOperationWrapper.FromOperation);
 
-    internal static ILiteralOperationWrapper? AsLiteral(this IOperation operation) =>
```

Comment:
This was added for that particular condition.
It's not used anywhere else anymore.

Reply:
Thanks for noticing

Reply:
Yeap - my bad.